Resources/config/security.xml

```diff
@@ -19,7 +19,8 @@
             <argument type="service" id="security.token_storage" />
             <argument type="service" id="security.user_checker" />
             <argument type="service" id="security.authentication.session_strategy" />
-            <argument type="service" id="service_container" />
+            <argument type="service" id="request_stack" />
+            <argument type="service" on-invalid="null" /> <!-- remember_me service -->
```

Comment:
I believe both are possible? I've seen many projects using a placeholder and replace it with `replaceArgument`.